engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/content-hosts-bulk-packages-modal.controller.js

```diff
@@ -134,10 +135,15 @@ angular.module('Bastion.content-hosts').controller('ContentHostsBulkPackagesModa
                 $scope.packageActionFormValues.remoteAction = 'package_' + action;
             }
 
-            if (selectedHosts.included.ids) {
+            if (selectedHosts.included.ids.length > 0) {
                 $scope.packageActionFormValues.hostIds = selectedHosts.included.ids.join(',');
             }
 
+            if (selectedHosts.allResultsSelected) {
+                $scope.scopedSearch = true;
```

Comment:
what about this?

```
$scope.allHostsSelected = selectedHosts.allResultsSelected;

if ($scope.allHostsSelected) {
...
}
```

That way you don't need to track `scopedSearch` separately and you can use it in the view as part of your conditional logic

Reply:
Good idea!